src/Grid.hpp

```diff
@@ -10,6 +10,7 @@
 #include <vector>
 #include <cmath>
 #include <functional>
+#include <random>
```

Comment:
Unused include (include it in the cpp file, nothing in this header needs it)

Reply:
fiixed